bleach/html5lib_shim.py

```diff
@@ -56,6 +56,8 @@
         list(constants.mathmlTextIntegrationPointElements)
     )
 ]
+# Add tags that aren't in html5lib.constants
+HTML_TAGS.extend(['abbr'])
```

Comment:
I did the `extend` here figuring we're going to probably add others in the future and might as well make it easier.